source-control/git.md

```diff
@@ -211,8 +211,77 @@ git stash
 git stash pop
 ```
 
-Or you can simply move the current state into a new branch:
+Or you can move the current state into a new branch:
 
 ```bash
 git stash branch <new_branch_to_save_changes>
 ```
+
+### Working with images, video and other binary content
+
+Avoid committing frequently changed binary files, such as large images, video or compiled code to your git repository. Binary content is not diffed like text content, so the repository will take longer and longer to clone.
```

Comment:
Great job removing superlatives above. Can simplify `longer and longer` to just `longer`, or make this more concrete by saying "Binary content is not diffed like text content, so cloning or pulling from the repository may pull each revision of the binary file".